meerk40t/core/elements.py

```diff
@@ -4518,13 +4518,19 @@ def classify(self, elements, operations=None, add_op_function=None):
                     break  # May only classify in Dots.
             if not was_classified:
                 if element.stroke is not None and element.stroke.value is not None:
-                    if Color.distance("red", element.stroke) < 90: # if element.stroke == Color("red"):
+                    if Color.distance_sq("red", element.stroke) <= 18825: # if element.stroke == Color("red"):
                         op = LaserOperation(operation="Cut", color=element.stroke)
+                    elif Color.distance_sq("black", element.stroke) <= 2304: # if element.stroke == Color("black"):
+                        op = LaserOperation(operation="Raster", color=element.stroke)
                     else:
                         op = LaserOperation(operation="Engrave", color=element.stroke)
                     add_op_function(op)
                     op.add(element, type="opnode")
                     operations.append(op)
+                elif element.stroke is None and element.fill is not None and element.fill.value is not None: # add operation for fill-only elements if not existing
+                    op = LaserOperation(operation="Raster", color=element.fill)
```

Comment:
@tatarize I realise that this code is somewhat simplistic.

It will deal with the reported bug whereby Inkscape Text->Path produces paths with no Stroke but a fill - and these do indeed need to be Raster operations.

But there is a common false-cut - which looks like a cut but is actually a very thin fill surrounded by a closed path without a stroke. These will be misclassified by this code. The options to fix this are either:

1. Use the colour to determine the Operation type exactly as in the code above; or
2. Analyse the path itself to determine whether the fill looks like a stroke (which seems way too difficult to me).

So I guess we should go for option 1. It is late and I am too tired to get my head around how to code this. Not sure if you want to take a shot at it, or whether you would like me to when I have the time to do it when I am fresh.